Events/src/main/java/com/github/swcraftlyon/meetup/infra/adapters/repositories/PlannerRepository.java

```diff
@@ -0,0 +1,26 @@
+package com.github.swcraftlyon.meetup.infra.adapters.repositories;
+
+import com.github.swcraftlyon.meetup.domain.entities.Event;
+import com.github.swcraftlyon.meetup.domain.IPlanner;
+import com.github.swcraftlyon.meetup.infra.dao.IEventDAO;
+import org.bson.Document;
+
+
+public class PlannerRepository implements IPlanner {
+
+    private final IEventDAO eventDAO;
+
+    public PlannerRepository(IEventDAO eventDAO) {
+        this.eventDAO = eventDAO;
+    }
+
+    @Override
+    public void record(Event event) {
+
+        Document document = new Document();
+        document.append("id", 1);
```

Comment:
et si on utilisait jamais de séquence :) 
https://www.clever-cloud.com/blog/engineering/2015/05/20/why-auto-increment-is-a-terrible-idea/